css-components/docs.css

```diff
@@ -0,0 +1,148 @@
+:root {
+    --link-hover-color: #9CA3AF;
+}
+
+body{
+    display: grid;
+    grid-template-rows: 100px 1fr 100px;
+    grid-template-columns: 1fr 5fr;
+    margin: var(--zero-percent);
+    font-family: 'Open Sans', sans-serif;
+}
+.doc-body {
+    grid-template-areas: "head head"
+                         "aside main"
+                         "footer footer";
+}
+
+.home-body{
+    grid-template-areas: "head head"
+                        "main main"
+                        "footer footer";
+}
+
+.header {
+    grid-area: head;
+    background-color: grey;
+    /* background-color: var(--main-color); */
+    color: white;
+    /* position: fixed; */
+    position: fixed;
+    width: 100%;
+    z-index: 1;
+}
+
+
+.aside {
+    grid-area: aside;
+    display: flex;
+    flex-direction: column;
+    /* align-items: center; */
+    justify-content: space-between;
+    /* max-width: var(--sidebar-width); */
+    margin: 2.5rem;
+    height: 66vh;
+    position: fixed;
+    font-weight: 700;
+    z-index: 2;
+    top: 4rem;
+
+
+}
+
+.aside-link {
+    text-decoration: none;
+    color: black;
+
+}
+
+.aside-link:hover {
+    color: var(--link-hover-color);
+    /* background-color: black;
+    color: white; */
+    /* border-radius: .5rem;
+    border: .12rem solid black; */
+}
+.home-img{
+    display: block;
+    max-width: 30%;
+    margin: auto;
+}
+
+.main {
+    grid-area: main;
+}
+
+.main-container {
+    margin: 1rem;
+}
+
+.main-container-heading {
+    margin: 3rem;
+}
+
+.main-container-components {
+    display: flex;
+    justify-content: space-evenly;
+    margin: 3rem;
+    padding: .5rem;
+    width: 90%;
+    /* flex-direction: row; */
+}
+
+.footer {
+    grid-area: footer;
+    display: block;
+    margin-top: 3rem;
+    margin-bottom: -1px;
+    height: 18vh;
+    width: 100%;
+    bottom: 0px;
+    background-color: var(--light-gray-color);
+}
+.footer >div{
+
+    margin: 1.5rem;
+}
+.footer ul{
+    display: flex;
+    flex-direction: row;
+    list-style-type: none;
+    justify-content: space-around;
+}
+.footer a{
+  color: black;
+}
```

Comment:
Use classes to target elements instead. Though here , since your html markup won't change frequently, so not a big deal. But still, using classes is better.